src/Http/Routing/src/DefaultEndpointConventionBuilder.cs

```diff
@@ -11,11 +11,13 @@ internal sealed class DefaultEndpointConventionBuilder : IEndpointConventionBuil
     internal EndpointBuilder EndpointBuilder { get; }
 
     private List<Action<EndpointBuilder>>? _conventions;
+    private List<Action<EndpointBuilder>>? _finalConventions;
 
     public DefaultEndpointConventionBuilder(EndpointBuilder endpointBuilder)
```

Comment:
Does anything still use this other than tests? I might have just forgotten to delete it when I moved all the old `Map` methods to `RouteHandlerBuilder`.

Reply:
Yep, this is only used in the tests.